### ISSUE TITLE
[FEATURE REQUEST] Flow-Icon for emergency generator

### ISSUE THREAD

**Issue by mariojungwirth** [flixlix/power-flow-card-plus#110]
There would still be room for an emergency power generator on the bottom left.
Could this still be implemented to display an emergency power generator flow.
The flow should then go to the grid. The unit would then be the power that the generator delivers.

**Comment by flixlix** [comment]
Shouldn't the generator only be producing towards the house? I think in that case, one of the individual devices could be used for that

**Comment by mariojungwirth** [comment]
Yes the generator only produce power to the house. But for the visibility is better the icon is on the left side. The generator is always on the same line as the utility.

**Comment by flixlix** [comment]
Can you photoshop or a quick mockup of how you would imagine this should look like?

**Comment by mariojungwirth** [comment]
I think ...

![image](https://user-images.githubusercontent.com/5375476/233348714-ddbec89c-7579-4d34-909f-a546f5781b56.png)

**Comment by mariojungwirth** [comment]
I Think...
![generator](https://user-images.githubusercontent.com/5375476/233349109-d56b382c-0134-4f87-aaff-88208f1443cd.png)

**Comment by flixlix** [comment]
Hmm ok, I'll add it to my todo-list, but I'm not convinced that this should flow to the grid...
